src/Events/Generator/API.php

```diff
@@ -153,6 +153,11 @@ public function generate_event() {
 
 		$event_id = tribe_create_event( $args );
 
+		// mark event, venue, and organizers as generated from tribe-cli
+		add_post_meta( $event_id, '_tribe_cli_generated', 1 );
```

Comment:
Set this as a String Variable on the class. `_tribe_cli_generated`

Reply:
Does that Boolean value really need to be that complicated?

Reply:
Oh maybe you're talking about the meta key string itself :)